cypress/integration/data-import/view-all.spec.js

```diff
@@ -0,0 +1,56 @@
+import ViewAllView from '../../support/fragments/data_import/viewAllView';
```

Comment:
think to rename to something more special. DataImportAllRecordsView for example. Or just define DataImport fragment with support of 2 view modes